model_compression_toolkit/exporter/README.md

```diff
@@ -46,14 +28,61 @@ import model_compression_toolkit as mct
 # Create a model
```

Comment:
it's not related to your changes, but can you correct line 39:
To export a TensorFlow model as a quantized model, it is necessary to first apply quantization to the model using MCT: